home/templates/pages/Contactus.html

```diff
@@ -19,23 +17,23 @@ <h1>Contact us</h1>
           <p>For further information, please contact us using the details below:</p>
           <div class="icon-text">
             <i class="fas fa-map-marker-alt"></i>
-            <span> 221 Burwood Hwy, Burwood VIC 3125</span>
+            <span><a href="https://g.co/kgs/abekme1" target="_blank">221 Burwood Hwy, Burwood VIC 3125</a></span>
           </div>
           <div class="icon-text">
             <i class="fas fa-phone"></i>
-            <span> 04******</span>
+            <span><a href="tel:+614******">04******</a></span> <!-- Phone number link -->
           </div>
           <div class="icon-text">
             <i class="fas fa-envelope"></i>
-            <span> hardhat@gmail.com</span>
+            <span><a href="mailto:hardhat@gmail.com">hardhat@gmail.com</a></span>
```

Comment:
We previously established that no one at Deakin actually has control over the email address used here,
could you replace it with a placeholder like the phone number

Reply:
Changed the text from hardhat@gmail.com to hardhat@example.com